brownie/network/account.py

```diff
@@ -259,7 +275,13 @@ def estimate_gas(self, to, amount, data=""):
                 return CONFIG['active_network']['reverting_tx_gas_limit']
             raise
 
-    def transfer(self, to, amount, gas_limit=None, gas_price=None, data=""):
+    def transfer(
+            self,
+            to: 'Accounts',
+            amount: int,
+            gas_limit: float = None,
+            gas_price: float = None,
+            data: str = "") -> 'TransactionReceipt':
```

Comment:
this looks so much cleaner than putting the `):` on the next line like a lonely frowny face :+1:

Reply:
PEP8 FTW!